rust/ql/lib/codeql/rust/Diagnostics.qll

```diff
@@ -48,7 +48,12 @@ class Diagnostic extends @diagnostic {
   string toString() { result = this.getMessage() }
 }
 
-/** A diagnostic relating to a particular error in extracting a file. */
+/** A diagnostic that is error severity. */
 class ExtractionError extends Diagnostic {
-  ExtractionError() { this.getTag() = "parse_error" }
+  ExtractionError() { this.getSeverity() = 40 }
```

Comment:
Don't use `30` and `40`, use `@diagnostic_warning` and `@diagnostic_error` instead.

Reply:
Yep that's much nicer, thanks.